app/qmlV2/component/MMPasswordEditor.qml

```diff
@@ -0,0 +1,72 @@
+/***************************************************************************
+ range.qml
+  --------------------------------------
+  Date                 : 2019
+  Copyright            : (C) 2019 by Viktor Sklencar
+  Email                : viktor.sklencar@lutraconsulting.co.uk
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Controls.Basic
+import QtQuick.Layouts
+import Qt5Compat.GraphicalEffects
+import ".."
+
+//import lc 1.0
+
+MMAbstractEditor {
+  id: root
+
+  /*required*/ property var parentValue: parent.value
+  /*required*/ property bool parentValueIsNull: parent.valueIsNull ?? false
+  /*required*/ property bool isReadOnly: parent.readOnly ?? false
+
+  required property string regexp
```

Comment:
Validation of password goes through c++, not here. This field will instead receive an error is password is not strong enough